app/src/main/java/com/simplecity/amp_library/saf/SafManager.java

```diff
@@ -221,7 +220,7 @@ private String getExtSdCardFolder(final File file) {
                 }
             }
         } catch (NoSuchMethodError e) {
-            Crashlytics.log("getExtSdCardPaths() failed. " + e.getMessage());
+
```

Comment:
We still need to log when exceptions occur. (`Log.e(getExtSdCardPaths() failed. " + e.getMessage())`)